### DISCUSSION TITLE
How do I determine if two or more Graphics overlap

### DISCUSSION THREAD

**Discussion by lykl** [pixijs/pixijs#9077] [Q&A] [answered]
How do I determine if two or more Graphics overlap

**Comment by bigtimebuddy** [comment, marked as answer]
There are a few ways to do this. The simplest is to compare the bounds and use the `intersects` method of Rectangle.

```ts
const graphics1 = new Graphics();
const graphics2 = new Graphics();

graphics1.getBounds().intersects(graphics2.getBounds());
```

Obviously this isn't perfect for all shapes. Pixi doesn't really have anything built-in to compare two polygons, but there are many other packages you can use (e.g., [polygon.js](https://www.npmjs.com/package/polygon), [polybooljs](https://www.npmjs.com/package/polybooljs) and others) which have this functionality. You'll just need to convert the `graphics.geometry` to these other libraries objects.

**Reply by lykl** [reply]
Thank you,it's just what I need
